src/server/components/dashboard/listsItems/helpers.ts

```diff
@@ -10,7 +10,7 @@ import { ListItemConfirmationPages, ListItemUrls } from "server/components/dashb
 import { ListItemJsonData } from "server/models/listItem/providers/deserialisers/types";
 import { dashboardRoutes } from "server/components/dashboard";
 
-export async function redirectIfUnauthorised(req: Request, res: Response, next: NextFunction): Promise<void> {
+export async function redirectIfUnauthorised(req: Request, _: Response, next: NextFunction): Promise<void> {
```

Comment:
This is because res isn't used. Just a style thing 😁

Reply:
eslint will whinge if a parameter or variable is unused. Mostly so a developer can stop and think, "do I need to pass in a thing that I don't need". 

But you can't remove `res` entirely here, otherwise the function would be `redirectIfUnauthorised(req, next)`...but express is expecting req, res, next. 

The convention is usually just to prefix the variable, i.e. `_res` rather than just `_`, so you don't clash with other unused params

Reply:
thanks for the explanation guys.  `_res` feels better to me